jinja2cli/cli.py

```diff
@@ -262,7 +263,7 @@ def render(template_path, data, extensions, strict=False):
                 extensions.append(ext)
 
     env = Environment(
-        loader=FileSystemLoader(os.path.dirname(template_path)),
+        loader=BaseLoader(),
```

Comment:
Won't this break if your template does an `{% include %}` now?

Reply:
You are right. Corrected